bundles/org.openhab.binding.shelly/README.md

```diff
@@ -406,8 +407,7 @@ The Dimmer should be calibrated using the Shelly App.
 
 |Group     |Channel      |Type     |read-only|Description                                                            |
 |----------|-------------|---------|---------|-----------------------------------------------------------------------|
-|control   |power        |Switch   |r/w      |Switch light ON/OFF                                                    |
```

Comment:
power channel is removed?

Reply:
yes, as with other lights power and brightness are controlled through the brightness channel. I don‘t like this, but it was requested in a previous review, now it‘s consistent with HUE, Wemo etc.